File: lambdas/death-enrichment/src/test/java/uk/gov/di/data/lep/PackageSettings.java
package uk.gov.di.data.lep;

import org.approvaltests.core.ApprovalFailureReporter;
import org.approvaltests.reporters.JunitReporter;
import org.approvaltests.reporters.AutoApproveWhenEmptyReporter;

public class PackageSettings {
    public static ApprovalFailureReporter UseReporter = new AutoApproveWhenEmptyReporter(new JunitReporter());
    public static ApprovalFailureReporter FrontloadedReporter = new AutoApproveWhenEmptyReporter(new JunitReporter());
    public static String UseApprovalSubdirectory = "approvals";
    public static String ApprovalBaseDirectory = "../resources";
}

File: platform/src/test/kotlin/PackageSettings.kt
package uk.gov.gdx.datashare

import org.approvaltests.core.ApprovalFailureReporter
import org.approvaltests.reporters.AutoApproveWhenEmptyReporter
import org.approvaltests.reporters.JunitReporter

class PackageSettings {
  var UseReporter: ApprovalFailureReporter = AutoApproveWhenEmptyReporter(JunitReporter())
  var FrontloadedReporter: ApprovalFailureReporter = AutoApproveWhenEmptyReporter(JunitReporter())
  var UseApprovalSubdirectory = "approvals"
  var ApprovalBaseDirectory = ""
}
